Tasks/InstallAppleProvisioningProfileV1/postinstallprovprofile.ts

```diff
@@ -1,13 +1,19 @@
 import path = require('path');
 import tl = require('vsts-task-lib/task');
 import sign = require('ios-signing-common/ios-signing-common');
+import os = require('os');
 
 import { ToolRunner } from 'vsts-task-lib/toolrunner';
 
 async function run() {
     try {
         tl.setResourcePath(path.join(__dirname, 'task.json'));
 
+        // Check platform is macOS since demands are not evaluated on Hosted pools
+        if (os.platform() !== 'darwin') {
```

Comment:
Do we want this to fail on 'post'?  This just deletes the file, and it can work on any OS.